src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.ts

```diff
@@ -3,6 +3,7 @@ import { VerificationFragmentType, Verifier } from "../../../types/core";
 import { OpenAttestationDidSignedDocumentStatusCode } from "../../../types/error";
```

Comment:
I dont think I've seen text covering the following (oal of the MR):

- DOCUMENT_STORE + another
- DID + another
- DNS-DID + another
- DNS-TXT + another

Reply:
Done. You can see latest commit on the unit tests https://github.com/Open-Attestation/oa-verify/pull/132/commits/3308b3edac14f2076c34a3e91255176512c46c40